README.md

```diff
@@ -1,6 +1,130 @@
+
+By taking into account additional context such as prevalent attacks and key protocols to the organization, the security team can develop a strategy that applies the right amount of per-channel risk mitigation based on the value of the data flowing through it.  For an organization, we call this “the port perspective”. 
+
+There are two vectors that all organizations should evaluate:
+
+ * A “wide enough, deep enough” protection strategy that involves both edge prevention and sophisticated detection of unusual behavior
+
+ * A deep inspection of key protocols using methods that can scale to the volume of data flowing across that channel
+
+While inspecting specific, unique flows of data that may be important for individual organizations, all organizations can realize significant risk reduction from analysis of network flows and DNS (domain name service) replies.
+
+**Open Network Insight**  by leveraging strong technology in both Big Data and Scientific Computing disciplines is a solution intended to support this strategy by focusing on “hard security problems” detecting events such as lateral movement, side-channel data escapes, insider issues, or stealthy behavior in general.  
+
+ 
+### **Telemetry**
+* Flows.
+* DNS (pcaps).
+
+### **Parallel Ingest Framework**
+* Open source decoders.
+* Load data in Hadoop.
+* Data transformation.
+
+
+### **Machine Learning**
+* Filter billion of events to a few thousands.
+* Unsupervised learning.
+
+### **Operational Analytics**
+* Visualization.
+* Attack heuristics.
+* Noise filter.
+
+
+## **Getting Started**
+
+ONI can be installed by following our installation manual. To get started, [check out the installation instructions in the documentation](https://github.com/Open-Network-Insight/open-network-insight/wiki).
+
+
+## If you want to get all the ONI code at one just clone it !
+        git clone --recursive https://github.com/Open-Network-Insight/open-network-insight.git
+
+## **Roadmap**
+
+TBD by RITU
+
+## **Documentation (Developer Guide)**
+
+ONI functionality is divided into different repositories, go to each repository for developer documentation:
+
+* [oni-ingest](https://github.com/Open-Network-Insight/oni-ingest)
+* [oni-ml](https://github.com/Open-Network-Insight/oni-ml)
+* [oni-oa](https://github.com/Open-Network-Insight/oni-oa)
+* [oni-setup](https://github.com/Open-Network-Insight/oni-setup)
+* [oni-nfdump](https://github.com/Open-Network-Insight/oni-nfdump)
+* [oni-lda-c](https://github.com/Open-Network-Insight/oni-lda-c)
+
+## **Community Support**
+
+Our Central repository for our Open Network Insight solution is found here. If you find a bug, have question or something to discuss please contact us:
+
+* [Create an Issue](https://github.com/Open-Network-Insight/open-network-insight/issues). if you need more details about how to create an issue in ONI please go to: [How to create and Issue](ISSUES.md)
```

Comment:
[How to create and Issue] --> [How to create an Issue]

I also think we should be more clear that "How to create an issue" is less of an educational document for those not aware, and more about guidelines for how we want the issue documented.

perhaps:
if you need more details about how to create an issue in ONI -->
For issue creation guidelines and how to create an issue for ONI